packages/pds/src/app-view/services/indexing/index.ts

```diff
@@ -107,7 +117,6 @@ export class IndexingService {
         .execute(),
     ])
     await Promise.all([
-      this.db.db.deleteFrom('record').where('did', '=', did).execute(),
```

Comment:
This moved to the PDS, since the record table really isn't governed by the app view.